helpers/discordAuth.ts

```diff
@@ -99,19 +95,21 @@ const getTokenFromRefreshToken = (
       grant_type: 'refresh_token',
       refresh_token
     })
-  }).then(r => r.json())
+  })
+  return await r.json()
```

Comment:
I don't think its right to do `return await <promise>` because the nature of `async` will return a promise already so we don't have to wait for the last promise to resolve before returning.

```js
return r.json()
``` 

should suffice. same with the rest.